Sources/LiveViewNative/ViewModel.swift

```diff
@@ -213,6 +213,6 @@ private extension URLComponents {
         else { return nil }
         // `+` is converted to a space, so query encode it.
         // Spaces are already converted to %20 by `percentEncodedQuery`
-        return query.replacingOccurrences(of: "+", with: "%2B")
+        return query.replacing("+", with: "%2B")
```

Comment:
is there a more comprehensive encoding rather than a single one off?

Reply:
I'm using the built-in percent encoding here: https://github.com/liveview-native/liveview-client-swiftui/pull/1449/files#diff-8a9b2e97113616f129b2f083e64768aeb40d0033ee8d7f20cc42f7940c63981bR212

But it doesn't cover the `+` character, because this is technically valid in query params. Another option would be to do a fully query encode of each value, only accepting a small character set (such as `.alphanumeric`) instead of `urlQueryAllowed`.